src/EnergyPlus/HVACManager.cc

```diff
@@ -2202,26 +2104,24 @@ void ResolveLockoutFlags(EnergyPlusData &state, bool &SimAir) // TRUE means air
     // METHODOLOGY EMPLOYED:
     // Checks if loop lockout flags are .TRUE.; if so, sets SimAirLoops to .TRUE.
 
-    auto &AirLoopControlInfo = state.dataAirLoop->AirLoopControlInfo;
-
     for (int AirLoopIndex = 1; AirLoopIndex <= state.dataHVACGlobal->NumPrimaryAirSys; ++AirLoopIndex) { // loop over the primary air loops
+        auto &airLoopControlInfo = state.dataAirLoop->AirLoopControlInfo(AirLoopIndex);
+
         // check if economizer ia active and if there is a request that it be locked out
-        if (AirLoopControlInfo(AirLoopIndex).EconoActive &&
-            (AirLoopControlInfo(AirLoopIndex).ReqstEconoLockoutWithCompressor || AirLoopControlInfo(AirLoopIndex).ReqstEconoLockoutWithHeating)) {
-            AirLoopControlInfo(AirLoopIndex).EconoLockout = true;
+        if (airLoopControlInfo.EconoActive &&
+            (airLoopControlInfo.ReqstEconoLockoutWithCompressor || airLoopControlInfo.ReqstEconoLockoutWithHeating)) {
+            airLoopControlInfo.EconoLockout = true;
             SimAir = true;
         }
     }
 }
 
-void ResetHVACControl(EnergyPlusData &state)
+void ResetHVACControl(const EnergyPlusData &state)
```

Comment:
[src/EnergyPlus/HVACManager.cc:2217]:(style),[constParameter],Parameter 'state' can be declared as reference to const

Reply:
@Myoldmopar I do have 2 branches but am reviewing the changes to see if I can avoid that step. This change added const to parameter state. However, inside this function is this and I am questioning if making state const is the correct thing to do. Maybe Linux debug doesn't like this change?

    for (auto &e : state.dataAirLoop->AirLoopControlInfo) {
        e.NightVent = false;